### DISCUSSION TITLE
How do I remove the standard Background?

### DISCUSSION THREAD

**Discussion by herbertfuchs** [react-navigation/react-navigation#10273] [Q&A]
Hey guys, so... I've been developing an Application in Android using react-native and I started to create the routes for it... But, the Navigator appears to come with a Background by nature, that doesn't respect the Background of the APP.

![BackgroundIssue](https://user-images.githubusercontent.com/82402110/149858340-2c5d8442-9548-49bf-8c09-f7fce97bb20c.png)

This is how I tried to remove the Background standard styles, but it didn't work... Any solution?

**Comment by raajnadar** [comment]
```
cardStyles: {
  backgroundColor: '#FFFFFF'
}
```

Will add white background? I don't thing transparent will work, because the root view will have some color that is specified from the native side.

What is your use case? Why transparent?

**Reply by herbertfuchs** [reply]
My application already have a background, just like the preview on the left side, but when I started using the navigation it became white.

![Captura de tela de 2022-01-19 14-17-52](https://user-images.githubusercontent.com/82402110/150181250-74040a64-9d6b-47b7-bebb-6256e134fa1c.png)

**Reply by herbertfuchs** [reply]
I tried to make the background transparent because I thought it could be the styles overlapping mine

**Reply by raajnadar** [reply]
Wrap the navigator with an image background component https://reactnative.dev/docs/imagebackground
